doc/api/addons.md

```diff
@@ -495,7 +495,7 @@ The functions available and how to use them are documented in
 ## Addon examples
 
 Following are some example addons intended to help developers get started. The
-examples make use of the V8 APIs. Refer to the online [V8 reference][v8-docs]
+examples use the V8 APIs. Refer to the online [V8 reference][v8-docs]
```

Comment:
```suggestion
examples utilize the V8 APIs. Refer to the online [V8 reference][v8-docs]
```

Reply:
What's the reasoning for this suggestion?

Reply:
They make use of them as a utility, no? (They find utility out of them.) Not blocking, just a thought. :)